lib/engine/game/g_18_eus/round/stock.rb

```diff
@@ -1,77 +1,21 @@
 # frozen_string_literal: true
 
 require_relative '../../../round/stock'
+require_relative 'bidbox_auction'
 
 module Engine
   module Game
     module G18EUS
       module Round
         class Stock < Engine::Round::Stock
+          include Engine::Game::G18EUS::Round::BidboxAuction
           attr_accessor :bids, :taken_loans, :paid_loans
 
           def setup
-            @game.setup_bidboxes
-            @stored_winning_bids = Hash.new { |h, k| h[k] = [] }
             @taken_loans = []
             @paid_loans = []
```

Comment:
I know this didn't change; however, these two fields aren't used here. Move to where they are used.

Reply:
Moved